cms/djangoapps/contentstore/views/course.py

```diff
@@ -458,12 +458,14 @@ def filter_ccx(course_access):
     return courses_list.values(), in_process_course_actions
 
 
-def _accessible_libraries_iter(user):
+def _accessible_libraries_iter(user, org=None):
     """
-    List all libraries available to the logged in user by iterating through all libraries
+    List all libraries available to the logged in user by iterating through all libraries.
+
+    If 'org' is present, only libraries from that org will be returned.
     """
     # No need to worry about ErrorDescriptors - split's get_libraries() never returns them.
-    return (lib for lib in modulestore().get_libraries() if has_studio_read_access(user, lib.location.library_key))
+    return (lib for lib in modulestore().get_libraries(org=org) if has_studio_read_access(user, lib.location.library_key))
```

Comment:
The upshot is that we *do* end up attaching the org to our query at a very low-level next to mongo: https://github.com/edx/edx-platform/blob/6d2468d0dcf782a7561ecc61946146d6fae2762f/common/lib/xmodule/xmodule/modulestore/split_mongo/mongo_connection.py#L475